docs-v2/integrations/all/front/connect.mdx

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This is not related to Connect UI.
Connect.mdx should only be used to guide end user, so only doc related to Connect UI

Reply:
Aah yes 🤦‍♂️, updated with the latest [commit](https://github.com/NangoHQ/nango/pull/3101/commits/bf4813f43df99624426ab34847dd5b9140aa8a4f). Thank you.